src/xunit.performance.api/ETWProfiler.cs

```diff
@@ -156,7 +304,13 @@ private static void SetPreciseMachineCounters(IEnumerable<ProviderInfo> provider
                 }
 
                 if (profileSourceIDs.Count > 0)
+                {
+                    // 
+                    // FIXME: This function changes the -pmcsources intervals machine wide.
```

Comment:
I do not think this is a new issue with the API. Can we change this in subsequent PR?